name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/util/LogoManager.java

```diff
@@ -30,7 +30,12 @@ private LogoManager()
 
     public Image getDefaultColumnImage(Object object, ClientSettings settings)
     {
-        Image logo = getLogoImage(object, settings);
+        return getDefaultColumnImage(object, settings, false);
+    }
+
+    public Image getDefaultColumnImage(Object object, ClientSettings settings, boolean disabled)
+    {
+        Image logo = getLogoImage(object, settings, disabled);
         return logo != null ? logo : getFallbackColumnImage(object);
```

Comment:
The issue is now: if the instrument has no logo (and hence the fallback image is used) then it is now the default image (either the active or the retired image). Basically, the "disabled" flag should also be applied to the fallback image (and probably the disabled fallback image should be cached somehow).

Reply:
Done a change and checked it in. Should now work as intended.